src/examples/button/SplitButtonDisabled.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { SplitButton, Button, ChevronButton } from '@zendeskgarden/react-buttons';
+import { Dropdown, Trigger, Menu, Item } from '@zendeskgarden/react-dropdowns';
+
+const Example = () => {
+  const [rotated, setRotated] = useState<boolean>();
+
+  return (
+    <Row>
+      <Col textAlign="center">
+        <SplitButton>
+          <Button disabled aria-disabled="true">
```

Comment:
Setting `disabled` [negates](https://codepen.io/stevef/post/short-note-on-the-disabled-attribute#also-note-5) any use of `aria-disabled`. The `aria-disabled` attribute may be removed here.